src/TimeKeeping.Core/Commands/ErfasseStechzeit.cs

```diff
@@ -61,7 +80,7 @@ protected override async Task<HandlerResult> ExecuteInternalAsync(ErfasseStechze
             var istNachgebucht = uhrabweichung > maxUhrabweichung;
             var istVorausgebucht = uhrabweichung < maxUhrabweichung;
 
-            var entry = new Zeiterfassung(
+            var entry = context ?? new Zeiterfassung(
                 command.ArbeitsplatzId,
                 command.Datum,
                 command.Stechzeit,
```

Comment:
Das knallt wenn DateTimeOffset nicht in UTC ist. Hab zumindest nicht gesehen, dass du es irgendwo transformierst.

Reply:
Ist das ein npgsql ding?

Reply:
Achso ja gilt nur für postgres.